include/websocket_handler.h

```diff
@@ -17,9 +17,35 @@
 
 #define WEBSOCKET_HOST_URI CONFIG_SERVER_WEBSOCKET_URI
 
+/*
+ * 0x00: this frame continues the payload from the last.
+ * 0x01: this frame includes utf-8 text data.
+ * 0x02: this frame includes binary data.
+ * 0x08: this frame terminates the connection.
+ * 0x09: this frame is a ping.
+ * 0x0a: this frame is a pong.
+ *
+ * A ping or pong is just a regular frame, but it's a control frame.
+ * Pings have an opcode of 0x9, and pongs have an opcode of 0xA.
+ * When you get a ping, send back a pong with the exact same Payload Data
+ * as the ping (for pings and pongs, the max payload length is 125).
+ * You might also get a pong without ever sending a ping; ignore this if it happens.
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers
+ */
+
+typedef enum websocket_op {
```

Comment:
I am not sure, but don't you need to assign value to each enum variable ?

Reply:
No need to assign values to all. In enum you can choose to assign value to one and next one will automatically +1 until new value is assigned.

Reply:
But, can you still assign them for better readability :) 

Not many people use enum and might get confused. Personally I prefer using macros

Reply:
Cool, I will assign values to them since there are limited Opcodes and possibly never going to grow, it is fine. 😃👍🏼

Reply:
done.